etc/ReplayOfflineConfiguration.py

```diff
@@ -1160,6 +1160,9 @@
 ignoreStream(tier0Config, "streamDQMRates")
 ignoreStream(tier0Config, "DQMPPSRandom")
 
+# Run only on Muon0 and Muon1 streams
+#specifyStreams(tier0Config, ["PhysicsMuon0","PhysicsMuon1"])
```

Comment:
We can provably remove these commented lines. They will be changed in any case in future replays.